qt/aqt/addons.py

```diff
@@ -246,11 +247,15 @@ def loadAddons(self) -> None:
             except AbortAddonImport:
                 pass
             except:
-                showWarning(
-                    tr.addons_failed_to_load(
-                        name=addon.human_name(),
-                        traceback=traceback.format_exc(),
-                    )
+                error = tr.addons_failed_to_load(
+                    name=addon.human_name(),
+                    traceback=traceback.format_exc(),
+                )
+                txt = f"<h1>{tr.qt_misc_error()}</h1><div style='white-space: pre-wrap'>{error}</div>"
```

Comment:
I think this may fail to render properly if the traceback happens to contains `<` or `>`? What about using the plain type instead, and passing qt_misc_error() in as the title= arg?

Reply:
I opted to just use html.escape here, like it's done in errors.py (which I tried to follow but didn't notice the html.escape call).

Reply:
That works too - thanks!